TransIT.API/Controllers/EmployeeController.cs

```diff
@@ -41,7 +44,8 @@ public async Task<IActionResult> Get([FromQuery] uint offset = 0, uint amount =
             }
             catch (Exception e)
             {
-                return StatusCode(500, e);
+                _logger.LogError(e, string.Empty);
```

Comment:
please do not log string.empty, log e.Messege

Reply:
Changed.